README.md

```diff
@@ -77,6 +77,56 @@ The following points can be customized in a packer variables file `variables.pkr
 - **Provisioning Scripts**: You can replace or modify the provisioning scripts located in the `./files/` directory.
   These scripts and files are copied and executed during the AMI creation process.
 
+## Amazon Linux Image Build
+
+This guide provides step-by-step instructions for building your own Amazon Linux AMI image.
```

Comment:
To build an AMI based on Amazon Linux you need to update the install_graphdb.sh script. 
This script installs necessary tools and configures the environment. Follow these steps to modify the script:
1. Open the install_graphdb.sh script in a text editor.
2.Replace the existing content between lines 12-36 with the following code:
<code>
3. Update variables.pkrvars.hcl values for source_ami_name_filter_arm64 and source_ami_name_filter_x86_64 as shown below:

Reply:
Done.